### PR TITLE
Added: Homepage ui and single video page ui

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added ui for homepage with categories section.
- Added single video page ui.
- Demo link --> https://deploy-preview-1--activa-react.netlify.app/

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Code looking good. You can go ahead !!

File: front-end/src/pages/home.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+								color2: '7dd3fc'
+							}}
+						/>
+						<VideoCard
+							videoCardData={{
+								image: cyberpunk,
+								title: 'Cyberpunk 2077',
+								brand: 'CD Projekt',
+								color1: 'f59e0b',
+								color2: 'fde68a'
```
Comment (Maianki) [inline front-end/src/pages/home.js:40]: Here you can use map instead of hardcoding VideoCard multiple times

**Review by Maianki — APPROVED** [review]
Everything looks good. Just has added one suggestion.

File: front-end/src/pages/home.js (hunk 1 above)
Comment (pratik-kumar-sahu) [reply front-end/src/pages/home.js:40]: Yes this will be removed when data comes from api

**Pull request merged**